website/blog/2022-08-18-new-release.md

```diff
@@ -0,0 +1,218 @@
+
+## Quality Enhancement
+
+Feature implementation is just the first part of product delivery. In Databend v0.8, the community introduced the concept of engineering quality, which evaluates the quality of Databend development in three dimensions: users, contributors, and community.
+
+### Reassuring users
+
+In order for users to use Databend with confidence, the community has added a lot of tests over the last three months, fetching stateless test sets from YDB and others, adding stateful tests for ontime, hits and other datasets, putting SQL Logic Test online to cover all interfaces, and enabling SQL Fuzz testing to cover boundary cases.
+
+Furthermore, the community has also gone live with [Databend Perf](https://perf.databend.rs/) to do continuous performance testing of Databend in production environments to catch unexpected performance regressions in time.
+
+### Make contributors comfortable
+
+Databend is a large Rust project that has been criticized by the community for its build time.
+
+To improve this issue and make contributors feel comfortable, the community went live with a highly configurable, specially tuned Self-hosted Runner to perform integration tests for PR and enabled several services or tools such as Mergify, mold, dev-tools, etc. to optimize the CI process.
+
+We also initiated a new plan to restructure the Databend project, splitting the original huge query crate into multiple sub-crates to avoid, as much as possible, the situation of changing one line of code and check execution for five minutes.
+
+### Keeping the community happy
+
+Databend is a contributor and participant in the open source community. During the development of v0.8, the Databend community established the principle of Upstream First, actively following and adopting the latest upstream releases, giving feedback on known bugs, contributing their own patches, and starting [Tracking issues of upstream first violation](https://github.com/datafuselabs/databend/issues/6926) to keep up with the latest developments.
+
+The Databend community is actively exploring integration with other open source projects and has already implemented integration and support for third-party drivers such as Vector, sqlalchemy, clickhouse-driver, etc.
+
+## Next Steps
+
+Databend v0.8 is a solid foundation release with a new Planner that makes it easier to implement features and make optimizations. In version 0.9, we expect improvements in the following areas.
+
+- Query Result Cache
+- JSON Optimization
+- Table Share
+- Processor Profiling
+- Resource Quota
+- Data Caching
+
+Please check the [Release proposal: Nightly v0.9](https://github.com/datafuselabs/databend/issues/7052) for the latest news~
+
+## Get going now!
+
+Visit the release log (link) and download the latest version (link) to learn more, and feel free to submit feedback using [Github Issues](https://github.com/datafuselabs/databend/issues) if you encounter problems!
```

Comment:
Update the link here

Reply:
Could you provide the link?

Reply:
Release Note & Download Link: https://github.com/datafuselabs/databend/releases/tag/v0.8.0-nightly